### PR TITLE
chore: required at least 1 approve before merge

### PR DESCRIPTION
**Pull request by jackwener**
## Which issue does this PR close?

Closes #.

## Rationale for this change

It is required that PR is approve to make Merge

### CODE REVIEW

**Comment by jackwener** [comment]
cc @liukun4515

**Review by liukun4515 — APPROVED** [review]

**Review by alamb — APPROVED** [review]
I think this is fine if other committers prefer this workflow. I have found it convenient on occasion to simply merge things prior to review to unblock CI pipelines but given the current activity of committers on DataFusion I think getting a review for such changes will be pretty fast.

**Comment by Dandandan** [comment]
> I think this is fine if other committers prefer this workflow. I have found it convenient on occasion to simply merge things prior to review to unblock CI pipelines but given the current activity of committers on DataFusion I think getting a review for such changes will be pretty fast.

Let's try and see if it's fast enough :)

**Review by Dandandan — APPROVED** [review]

**Pull request merged**
